jb/itch5/order_book.hpp

```diff
@@ -42,7 +53,15 @@ typedef std::pair<price4_t, int> half_quote;
  * the total quantity available at that price?
  * - What is the best offer (lowest price of SELL orders) and what is
  * the total quantity available at that price?
+ *
+ * This is a template class.
+ * @param book_type define data structure type of price book,
+ * both sides buy and sell. Allowed values are:
+ * - jb::itch5::map_price
```

Comment:
In general, one does not want to specify the list of types that a template class can be used with.  One specifies the requirements on the template parameter, if needed one can say "just model it after 'foo_type'".

The reason is that template classes are the analogous of functions: you describe the valid set of inputs, you do not enumerate them.